packages/nimble-components/src/anchor-tab/styles.ts

```diff
@@ -1,14 +1,124 @@
 import { css } from '@microsoft/fast-element';
-import { styles as tabStyles } from '../patterns/tab/styles';
+import { display } from '@microsoft/fast-foundation';
+import { focusVisible } from '../utilities/style/focus';
+import {
+    bodyDisabledFontColor,
+    bodyFontColor,
+    borderHoverColor,
+    borderWidth,
+    buttonLabelFont,
+    controlHeight,
+    fillHoverColor,
+    fillHoverSelectedColor,
+    smallDelay,
+    standardPadding
+} from '../theme-provider/design-tokens';
 
 export const styles = css`
-    ${tabStyles}
+    ${display('inline-flex')}
+
+    :host {
+        position: relative;
+        box-sizing: border-box;
+        font: ${buttonLabelFont};
+        height: ${controlHeight};
+        color: ${bodyFontColor};
+        align-items: center;
+        justify-content: center;
+        cursor: pointer;
+        --ni-private-active-indicator-width: 2px;
+        --ni-private-focus-indicator-width: 1px;
+        --ni-private-indicator-lines-gap: 1px;
+        --ni-private-focus-indicator-inset-width: 3px;
+    }
+
+    :host(:hover) {
+        background-color: ${fillHoverColor};
+    }
+
+    :host(:hover[aria-selected='true']) {
+        background-color: ${fillHoverSelectedColor};
+    }
+
+    :host(:active) {
+        background: none;
+    }
+
+    :host([disabled]) {
+        cursor: default;
+        color: ${bodyDisabledFontColor};
+        background: none;
+    }
+
+    slot:not([name]) {
+        display: block;
+        padding: calc(${standardPadding} / 2) ${standardPadding}
+            calc(${standardPadding} / 2 - ${borderWidth});
+    }
+
+    a::before {
+        content: '';
+        position: absolute;
+        bottom: calc(
+            var(--ni-private-active-indicator-width) +
+                var(--ni-private-indicator-lines-gap)
+        );
+        width: 0px;
+        height: 0px;
+        border-bottom: ${borderHoverColor}
+            var(--ni-private-focus-indicator-width) solid;
+        transition: width ${smallDelay} ease-in;
+    }
+
+    @media (prefers-reduced-motion) {
+        a::before {
+            transition-duration: 0.01s;
```

Comment:
Any reason to not just set this to 0 seconds? While I do see some other cases where we use the '.01s' value, most `trasition-duration` values are set to '0s'. I'm not sure if there's even a noticeable difference between the two.

Reply:
It's copied from the tab styling. I'll set them both to 0.